### ISSUE TITLE
git config user and email not set

### ISSUE THREAD

**Issue by harleypig** [plandex-ai/plandex#8]
This may or may not be related to running this under docker compose.
```bash
$ pdx load bin/* lib/*
🚨 Failed to load context
  → Error committing changes
    → Error committing files to git repository for dir
      → /root/plandex-server/orgs/0368f56f-0794-4a83-8324-0392b07e4338/plans/3b6546fe-6408-4d3b-bfa3-b62df1cb3d3f, err
        → Exit status 128, output
          → Author identity unknown

*** Please tell me who you are.

Run

  git config --global user.email "you@example.com"
  git config --global user.name "Your Name"

to set your account's default identity.
Omit --global to set the identity only in this repository.

fatal
            → Unable to auto-detect email address (got 'root@224f0a0ed4c9.(none)')
```

**Comment by harleypig** [comment]
I'm not sure if this needs a new issue or not. I'll add it here and create a new one if you think it needs it.

I noticed the path is not what I expected.
```
      → /root/plandex-server/orgs/0368f56f-0794-4a83-8324-0392b07e4338/plans/3b6546fe-6408-4d3b-bfa3-b62df1cb3d3f, err
```
Shouldn't this be `/plandex-server` not `/root/plandex-server`? I don't think this is how I'm building the image.
```
$ docker inspect --format='{{json .Mounts}}' app-plandex-server-1 | jq '.' ; sudo ls -l $HOME/projects/plandex-data/plandex-data
[
  {
    "Type": "bind",
    "Source": "/home/harleypig/projects/plandex-data/plandex-data",
    "Destination": "/plandex-server",
    "Mode": "rw",
    "RW": true,
    "Propagation": "rprivate"
  }
]
[sudo] password for harleypig: 
Executing: sudo ls -l /home/harleypig/projects/plandex-data/plandex-data 
total 0
```

**Comment by danenania** [comment]
@harleypig Yeah would you mind creating a separate issue for the `/root/plandex-server` thing. It should indeed be `/plandex-server` by default. Not sure where the `/root` is coming from.

The git user/email issue should be fixed in `server/0.8.1`
